### PR TITLE
LPS-153804 - Change in Workflow Constants

### PR DESCRIPTION
**Pull request by Gabriel422-hue**
Related Issue: https://issues.liferay.com/browse/LPS-153804

Hi Sam! This PR, corrects the user's problem, being able to include new attachments, after being rejected by its creator, even if he has already sent it. Questions can call, thank you!

### CODE REVIEW

**Comment by SamZiemer** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormInstanceSubmissionLimitStatusUtil.java [hunk 1]
```diff
@@ -69,14 +69,10 @@ public static boolean isSubmissionLimitReached(
...
-					WorkflowConstants.STATUS_DRAFT
+					WorkflowConstants.ACTION_PUBLISH
 		).findFirst();
 
-		if (optional.isPresent()) {
-			return true;
-		}
-
-		return false;
+		return optional.isPresent();
```
Comment (SamZiemer) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormInstanceSubmissionLimitStatusUtil.java:75]: Was this change intended? Based on the commit message this maybe should not have been added to the commit.

File: modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/exporter/LayoutsExporterImpl.java [hunk 2]
```diff
@@ -123,9 +123,25 @@ public File exportLayoutPageTemplateEntries(
 			long[] layoutPageTemplateEntryIds, int type)
 		throws Exception {
 
-		return _exportLayoutPageTemplateEntries(
```
Comment (SamZiemer) [inline modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/exporter/LayoutsExporterImpl.java:126]: This commit does not appear to be related to the issue in LPS-153804, what is the reason for this change? In addition, though you are checking what type of LayoutPageTemplateEntry "type" is, nothing changes regardless of what type it is, so I don't think this will work.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormInstanceSubmissionLimitStatusUtil.java (hunk 1 above)
Comment (Gabriel422-hue) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormInstanceSubmissionLimitStatusUtil.java:75]: That, that change was the IDE, which ended up suggesting. It shouldn't be there, I'll update the commit.

File: modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/exporter/LayoutsExporterImpl.java (hunk 2 above)
Comment (Gabriel422-hue) [reply modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/exporter/LayoutsExporterImpl.java:126]: I think these files were sent, along with the commit, because your Repo was out of date

**Comment by SamZiemer** [comment]
ci:close
